### PR TITLE
Trim trailing slashes. Fixes #102

### LINKED ISSUE #102: Trim URL Trailing Slashes
**Is your feature request related to a problem? Please describe.**
Environment URLs containing trailing slashes cause the tool to fail as it'll reach out to `https://abc123.live.dynatrace.com//api`

**Describe the solution you'd like**
Trim any trailing slashes

### PR DESCRIPTION
**Pull request by Dynatrace-Adam-Gardner**

### CODE REVIEW

File: pkg/environment/environment.go [hunk 1]
```diff
@@ -94,6 +94,8 @@ func newEnvironment(id string, properties map[string]string) (Environment, error
 }
 
 func NewEnvironment(id string, name string, group string, environmentUrl string, envTokenName string) Environment {
+	environmentUrl = strings.TrimSuffix(environmentUrl, "/")
```
Comment (didiladi) [inline pkg/environment/environment.go:97]: Cool, awesome! :1st_place_medal: 

One minor thing: Can you add a unit test for that in `pkg/environment/environment_test.go`?

Comment (Dynatrace-Adam-Gardner) [reply pkg/environment/environment.go:97]: I need a badge. My first ever Go unit test. :) Hope that's all good! Merry Christmas.

Comment (didiladi) [reply pkg/environment/environment.go:97]: Cool, the test looks awesome! Thank you! :rocket: Merry Christmas and a Happy new Year! :christmas_tree:

**Comment by didiladi** [comment]
The canges look good, just running the integation tests is missing here.

@Dynatrace-Adam-Gardner please mark your repo as non-private. The github action for triggering the integration tests fails miserably because it can't find the repo.

**Review by didiladi — APPROVED** [review]

**Review by UnseenWizzard — APPROVED** [review]

**Comment by Dynatrace-Adam-Gardner** [comment]
I've obviously screwed something up here :( My repo is [here](https://github.com/Dynatrace-Adam-Gardner/dynatrace-monitoring-as-code).

If it's easier, I'll delete this PR and re-open?

**Comment by UnseenWizzard** [comment]
Hi @Dynatrace-Adam-Gardner sorry for the late reply. 
I have no idea what exactly is wrong with this PR - it shows your source repo as "unknown repo", did you move or rename your repo ? 

It will probably be indeed easiest if you close this PR and open a new one from your repo!

**Pull request closed without merging**